OracleSOA/samples/12212-osb-compact-domain/container-scripts/create-osb-domain.py

```diff
@@ -0,0 +1,69 @@
+domain_name  = os.environ.get("DOMAIN_NAME", "base_domain")
```

Comment:
I believe the shebang line indicates what executable to run, so if it had it, wouldn't that break the way it works?  I assume that is a WLST script meant to be run in Jython, so specifying `/usr/bin/python` would probably fail, correct?